lib/immobilienscout24/api.rb

```diff
@@ -10,6 +10,7 @@
 require "immobilienscout24/api/attachment"
 require "immobilienscout24/api/contact"
 require "immobilienscout24/api/gis"
+require "lib/immobilienscout24/api/expose"
```

Comment:
This line seems to cause an error in the specs.
It should be `"immobilienscout24/api/expose"`